Automattic/wp-calypso
# Add playwright and dependencies to test/e2e and instruct playwright to install browser binaries into node_modules folder.

#### Changes proposed in this Pull Request

* adds the Playwright package as a dependency for `test/e2e` in preparation for a migration proof of concept.
* add environment variable `PLAYWRIGHT_BROWSERS_PATH=0` to install binaries into the `node_modules` directory, which persists across steps.

#### Testing instructions

Ensure that builds do not break.
Ensure that when the environment is spun up, the additional installation of Playwright does not break things.

#### Notes

[This TeamCity build](https://teamcity.a8c.com/buildConfiguration/calypso_RunCalypsoE2eDesktopTests/5978104?buildTab=log&focusLine=2) demonstrates a script that launches an instance of Chromium, opens the login page then shuts down. I have removed the script from this PR.
